src/components/metadata/metadata-code.js

```diff
@@ -6,19 +6,26 @@ import yaml from 'highlight.js/lib/languages/yaml';
 import modifiers from '../../utils/modifiers';
 import './styles/metadata-code.css';
 
+let javascript;
 hljs.registerLanguage('python', python);
 hljs.registerLanguage('yaml', yaml);
 
 /**
  * A highlighted code panel
  */
 export const MetaDataCode = ({
+  dataIsRandom,
+  sidebarVisible,
   visible = true,
   value = '',
-  sidebarVisible,
 }) => {
   const codeRef = useRef();
 
+  if (dataIsRandom && !javascript) {
+    javascript = require('highlight.js/lib/languages/javascript');
```

Comment:
Interesting what's the reasoning behind late require here rather than doing it at the start of the file?